src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -158,7 +158,11 @@ private static String getCommonConfig(final TravelOptions travelOptions) throws
             }
 
             if (travelOptions.getMaxSnapDistance() != null){
-                JSONBuilder.append(config, MAX_SNAP_DISTANCE, travelOptions.getMaxSnapDistance());
```

Comment:
why removing the snap distance option?

Reply:
oups I didn't want that. Thanks for the catch!